spotless and markdown

We need check only project markdown files.
Generated files from target should be excluded.

Reproduce, eg in tidy-maven-plugin execute site build twice:

```
mvn site

mvn site 
```

we have:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-site-plugin:3.12.1:site (default-site) on project tidy-maven-plugin: failed to get report for org.apache.maven.plugins:maven-plugin-report-plugin: Failed to execute goal com.diffplug.spotless:spotless-maven-plugin:2.43.0:check (spotless-check) on project tidy-maven-plugin: The following files had format violations:
[ERROR]     target/site/apidocs/legal/jquery.md
[ERROR]         @@ -1,6 +1,7 @@
[ERROR]          ##·jQuery·v3.6.1
[ERROR]          
[ERROR]          ###·jQuery·License
[ERROR]         +
[ERROR]          ```
....
```

--- a/pom.xml
+++ b/pom.xml
@@ -600,6 +600,9 @@
               <includes>
                 <include>**/*.md</include>
               </includes>
+              <excludes>
+                <excludes>target/**</excludes>
+              </excludes>
               <flexmark />
             </markdown>
             <upToDateChecking>

